packages/@okta/vuepress-theme-prose/util/coveo/endpoint.js

```diff
@@ -1,27 +1,43 @@
 import axios from "axios";
+import jwt_decode from "jwt-decode";
+import moment from "moment";
 
 const TOKEN_ENDPOINT =
   "https://wt1ugse0be.execute-api.us-west-2.amazonaws.com/prod/token?site=developer";
 const COVEO_PIPELINE = "oktaproduction9ounvcxa";
 const COVEO_ENDPOINT = "https://platform.cloud.coveo.com/rest/search";
 
-const _getToken = async () => {
-  const {
-    data: { token }
-  } = await axios.get(TOKEN_ENDPOINT);
+const _getToken = () => {
+  const token = localStorage.getItem("coveo_token");
+  if (!token) return;
 
-  localStorage.setItem("coveo_token", token);
-  return token;
+  // Invalidate token if expired
+  const decoded = jwt_decode(token);
+  const now = moment(Date.now() / 1000);
```

Comment:
Is this the same as `moment()`?

Reply:
Was converting this one to match `exp`'s format but now doing it the other way around with `moment.unix(exp)` which does `* 1000` internally